src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/MsftToSbSdk.diff

```diff
@@ -546,4 +563,4 @@ index ------------
 -./sdk/x.y.z/zh-Hant/Microsoft.VisualStudio.Coverage.IO.resources.dll
  ./sdk/x.y.z/zh-Hant/MSBuild.resources.dll
  ./sdk/x.y.z/zh-Hant/System.CommandLine.resources.dll
- ./shared/
\ No newline at end of file
+ ./shared/
```

Comment:
I believe the blank extra line at the end of the file will cause a diff to be detected. The content should be exactly the same as the file that was included in the pipeline artifacts.

Reply:
I believe so too. I've tried disabling the setting in my IDE that adds the newline on `save` of the file, and I tried committing a change via github.com to remove this space, but it always gets added back automatically. Not sure what else I can do to try and remove it.